### DISCUSSION TITLE
Turbulence Intensity Parameter

### DISCUSSION THREAD

**Discussion by seltzera** [NatLabRockies/floris#827] [Q&A]
Does FLORIS take the streamwise turbulence intensity or the total turbulent kinetic energy, when the ambient turbulence is inputted in a Gaussian wake model? 

For instance, I am thinking of the examples where gch.yaml is the input file, where a parameter is:
  "#The level of turbulence intensity level in the wind.
       turbulence_intensity: 0.06"
  
Here, is the turbulence_intensity parameter the streamwise or the total from all directions?

**Comment by misi9170** [comment]
Hi @seltzera , FLORIS takes the streamwise turbulence intensity. The turbulence intensity model used by most models in FLORIS is the Crespo-Hernandez model, originally described [here](https://www.sciencedirect.com/science/article/pii/016761059500033X), if you are looking for more details about how the ambient TI is used and how it contributes to wake expansion.
